src/main/java/com/github/retro_game/retro_game/service/impl/BodyCreationServiceImpl.java

```diff
@@ -161,7 +161,7 @@ private int generatePlanetDiameter(int position) {
     double x = Math.abs(8 - position);
     double mean = 200.0 - 10.0 * x;
     double sd = 60.0 - 5.0 * x;
-    double numFields = mean + sd * random.nextGaussian();
+    double numFields = mean + sd * (-0.25 + Math.random() * (1 - -0.25));
```

Comment:
This is the formula to convert from the standard distribution to the normal distribution for the given planet position (`mean`, `sd`):
https://sphweb.bumc.bu.edu/otlt/mph-modules/bs/sas/sas4-onesamplettest/SAS4-OneSampleTtest4.html.

It would be better if the planet size is adjusted in the `mean` and `sd` variables instead of in this formula.

If I'm not mistaken, this calculates the number of fields as the `mean` would be `mean - 0.25*sd` and `sd` would be `1.25*sd`, so on average the planets will be smaller, but the spread is higher.

Reply:
Gaussian (normal) distribution has unlimited range ((-∞, ∞)). So in theory you may roll a negative size (corrected one line later by `max`) or very large one (`sqrt` is taken, but huge numbers would remain huge).
In practice the [values are limited](https://stackoverflow.com/questions/54712600/what-is-the-true-maximum-and-minimum-value-of-random-nextgaussian), though.

The proposed change, with standard uniform distribution (`Math.random`) restricts the random part to (-0.25, 1), which is arguably good (at least the restriction, not necessarily the range). The mean value also shifts: from 0 in case of standard normal distribution (`nextGaussian`) to 0.375.

Also, `ThreadLocalRandom` is replaced by standard `Math.random`.